client/android/app/src/main/AndroidManifest.xml

```diff
@@ -3,28 +3,39 @@
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.VIBRATE" />
-    <uses-sdk android:minSdkVersion="16" android:targetSdkVersion="22" />
-    <application android:name=".MainApplication" android:allowBackup="true" android:label="@string/app_name" android:icon="@mipmap/ic_launcher" android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity" android:label="@string/app_name" android:configChanges="keyboard|keyboardHidden|orientation|screenSize" android:launchMode="singleTop" android:windowSoftInputMode="adjustResize">
-            <intent-filter>
-                <action android:name="android.intent.action.MAIN" />
-                <category android:name="android.intent.category.LAUNCHER" />
-            </intent-filter>
-        </activity>
-        <service android:name="io.invertase.firebase.messaging.RNFirebaseMessagingService">
-            <intent-filter>
-                <action android:name="com.google.firebase.MESSAGING_EVENT" />
-            </intent-filter>
-        </service>
-        <service android:name="io.invertase.firebase.messaging.RNFirebaseInstanceIdService">
-            <intent-filter>
-                <action android:name="com.google.firebase.INSTANCE_ID_EVENT" />
-            </intent-filter>
-        </service>
-        <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
-        <meta-data android:name="com.google.android.geo.API_KEY" android:value="AIzaSyCsYLUZTuw9bEjK-Ps4W42HiwvF7gE8rgw" />
-        <meta-data android:name="com.bugsnag.android.API_KEY" android:value="08136489878ec2af26c61e6dd7acdd98" />
-        <meta-data android:name="firebase_analytics_collection_deactivated" android:value="true" />
-        <meta-data android:name="google_analytics_adid_collection_enabled" android:value="false" />
+    <application
```

Comment:
I hate touching this file because react native tries to manage it. Can you do a quick google to see if these settings can be managed in build.gradle or something instead rather than here?

Reply:
Everything ive read does it directly to the core files unfortunately.  I cant find any middleware to let react-native take care of it

Reply:
unless you want this kinda of stuff - https://blog.bam.tech/developper-news/change-your-react-native-app-icons-in-a-single-command-line

Reply:
not worth